packages/kmod-6.12-amdgpu/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "kmod-6_12-amdgpu"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[package.metadata.build-package]
+package-name = "kmod-6.12-amdgpu"
+
+[lib]
+path = "../packages.rs"
+
+[[package.metadata.build-package.external-files]]
+url = "https://repo.radeon.com/amdgpu/30.10.2/el/10/main/x86_64/amdgpu-dkms-6.14.14-2226257.el10.noarch.rpm"
```

Comment:
Can we try the newer version: https://repo.radeon.com/amdgpu/30.20/el/10/main/x86_64/?